### DISCUSSION TITLE
How to Update Facebook SDK for iOS 14

### DISCUSSION THREAD

**Discussion by Haruna54** [community/community#47833] [Other Feature Feedback, Questions, & Ideas]
<img width="960" alt="2023-02-17 (2)" src="https://user-images.githubusercontent.com/125680650/219804689-74de1c0f-c139-4769-83f5-066943087c6a.png">
I have been searching for a Meta developer regarding the installation of Facebook SDK for iOS version 14.1.0. This has been frustrating for five days. In order to receive instructions, I was instructed to send an email to my developer. Unfortunately, I do not have any. My lack of knowledge in all of this makes it impossible for me to navigate this on my own. Through this medium, I am appealing to Meta Developers alongside all community members with ideas for solutions to my problem. Please accept my gratitude in advance!

**Comment by queenofcorgis** [comment]
Hi @Haruna54,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Meta developers. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by Haruna54** [reply]
Thank you so much
